tools/scenario-player/scenario_player/utils.py

```diff
@@ -71,8 +71,7 @@ def getlines(self, start, stop=None):
 
 
 class ConcatenableNone:
-    def __radd__(self, other):
```

Comment:
AFAIK the `__radd__` needs to stay

Reply:
when we have this, then my terminal always flickers, why do we need the `__radd__`?

Reply:
See my explanation to Pauls question above. I don't get how this works for you without the `__radd__`. 

Which terminal are you using?

Reply:
For the scenario player I use the terminal in PyCharm atm